event_email_reminder/models/event.py

```diff
@@ -10,17 +11,31 @@
 class EventEvent(models.Model):
     _inherit = 'event.event'
 
+    @api.model
+    def _send_event_template(self, events, template, partner_ids):
+        ctx = self.env.context.copy()
+        ctx.update({
+            'events': events.sorted(lambda x: x.date_begin),
+        })
+        for partner_id in partner_ids:
+            ctx.update({
+                'lang': self.env['res.partner'].browse(partner_id).lang,
```

Comment:
This is still not working for event names, as you are putting the context for the template, not for the event itself.